### PR TITLE
Add Starting Documentation

### PR DESCRIPTION
**Pull request by fmoletta**

### CODE REVIEW

File: docs/Starting Documentation.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Cairo Vm
+===
+
+# Must Read
+
+* From Cairo Documentation: [How Cairo Works](https://www.cairo-lang.org/docs/how_cairo_works/index.html#how-cairo-works)
+
+* [CairoVm code](https://github.com/starkware-libs/cairo-lang/tree/b614d1867c64f3fb2cf4a4879348cfcf87c3a5a7/src/starkware/cairo/lang/vm)
```
Comment (martincastelli) [inline docs/Starting Documentation.md:8]: Please, change the link to use the master branch:
https://github.com/starkware-libs/cairo-lang/tree/master/src/starkware/cairo/lang/vm

**Review by JuanLGomez — APPROVED** [review]
This is ok

**Review by martincastelli — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Cairo Vm
+
+## Must Read
+
+* From Cairo Documentation: [How Cairo Works](https://www.cairo-lang.org/docs/how_cairo_works/index.html#how-cairo-works)
+
+* [CairoVm code](https://github.com/starkware-libs/cairo-lang/tree/master/src/starkware/cairo/lang/vm)
+
+## Complimentary Reading
```
Comment (JuanLGomez) [inline docs/README.md:9]: It should say "complementary".

**Review by JuanLGomez — CHANGES_REQUESTED** [review]
Minor spelling

**Pull request merged**